internal/commands/create.go

```diff
@@ -224,6 +226,62 @@ func getRegoFilePaths(path string) ([]string, error) {
 	return regoFilePaths, nil
 }
 
+func parsePolicies(policyPaths []string, libraryPaths []string) ([]*regoPolicy, error) {
+	policies, err := loadPolicies(policyPaths)
+	if err != nil {
+		return nil, err
+	}
+	libraries, err := loadPolicies(libraryPaths)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, p := range policies {
+		if len(p.policy.Imports) > 0 {
+			for _, i := range p.policy.Imports {
+				library := getLibrary(libraries, i.Path.String())
+				if library == nil {
+					return nil, fmt.Errorf("imported library %s not found", i.Path.String())
+				}
+
+				// We read the file again from disk to perserve the formatting of the policy
+				// The OPA parser removes a lot of the nice syntax sugar that makes it easier for us to read
+				// ---
+				// We just read the library a few milliseconds ago, assuming errors won't happen on the second read
+				data, _ := ioutil.ReadFile(library.path)
```

Comment:
nit: I too like to live dangerously :) though we should always check for errors and return if we found one